Tests/XCUITests/ActivityStreamTest.swift

```diff
@@ -11,32 +11,21 @@ let allDefaultTopSites = ["Facebook", "YouTube", "Amazon", "Wikipedia", "Twitter
 class ActivityStreamTest: BaseTestCase {
     let TopSiteCellgroup = XCUIApplication().cells[AccessibilityIdentifiers.FirefoxHomepage.TopSites.itemCell]
 
-    let testWithDB = ["testActivityStreamPages", "testTopSites2Add"]
+    let testWithDB = ["testTopSites2Add", "testTopSitesRemoveAllExceptDefaultClearPrivateData"]
```

Comment:
It looks like `testTopSitesRemoveAllExceptDefaultClearPrivateData()` needs the database to be loaded because the test is looking mozilla.org to be one of the page visited.